tango_simlib/tango_sim_generator.py

```diff
@@ -459,6 +461,9 @@ def get_parser_instance(sim_datafile):
     return parser_instance
 
 def configure_device_model(sim_data_file=None, test_device_name=None):
+    return configure_device_models(sim_data_file, test_device_name)
```

Comment:
This isn't quite the same as before - it used to return a single model object instance.  I was thinking something like this:
```suggestion
    models = configure_device_models(sim_data_file, test_device_name)
    if len(models) == 1:
        return models.values()[0]
    else:
        raise RuntimeError('Single model expected, but found {} models in {}'
                           .format(len(models), sim_data_file))
```

Reply:
Discussed with @kmadisa - new behaviour is preferred.  `get_tango_device_server` now accepts a dict, so return the dict on line 466 instead of the value.  This keeps things largely compatible for existing users.